src/GeositeFramework/plugins/mapview-sceneview-toggle/main.js

```diff
@@ -0,0 +1,30 @@
+define(
+    ["dojo/_base/declare", "framework/PluginBase"],
+    function (declare, PluginBase) {
+        return declare(PluginBase, {
+            toolbarName: "MapView-SceneView Toggle",
+            fullName: "Toggle between 2-D MapView and 3-D SceneView",
```

Comment:
Not sure it's meaningful to expose the class names to the user here, probably better to be "Toggle between 2D and 3D mode"